layers/+emacs/org/packages.el

```diff
@@ -588,6 +590,12 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (kbd "M-SPC") 'spacemacs/org-agenda-transient-state/body
       (kbd "s-M-SPC") 'spacemacs/org-agenda-transient-state/body)))
 
+(defun org/init-org-wild-notifier ()
+  (use-package org-wild-notifier
+    :init
```

Comment:
I think this package should be deferred.

Reply:
Ah don't know why I left it out, but should be there indeed. I have pushed the update. Thanks again for checking and feedback!